src/components/navMenu/mobileNavMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import { dcNavLinks } from "@/data/dcNavLinks";
+import {
+  Text,
+  Link,
+  Icon,
+  Button,
+  useCloseDropDown,
+  List,
+} from "@nypl/design-system-react-components";
+import React, { useRef, useState } from "react";
+import FocusLock from "@chakra-ui/focus-lock";
+import { Box, chakra } from "@chakra-ui/react";
+import { useScrolled } from "@/hooks/useScrolled";
+
+const listItems = dcNavLinks.map(({ href, text }) => (
+  <Link
```

Comment:
For lists, the `ul` or `ol` elements should have `li` children. In this case, wrap the `Link` components in `<li>`s.

Reply:
Passing this to `listitems` does this automatically, so that creates nested `<li>` elements unnecessarily

Reply:
Yes, I forgot that's how that works! :face-palm: (there's no face palm emoji)